### DISCUSSION TITLE
How to render a flet app inside an already running flet app?

### DISCUSSION THREAD

**Discussion by frinkifail** [flet-dev/flet#887] [Q&A]
This is my file structure
- main.py
- another_app.py

inside both of them are flet apps

the main one just has a text saying "Hello"
and the other one has a text saying "Another file!"

how can I get the another_app to render within the main one without having to open another window?

**Comment by FeodorFitsner** [comment]
How about https://flet.dev/docs/controls/fletapp?

**Reply by frinkifail** [reply]
how do I use it?
the docs doesn't explain that much on how to embed it...

**Reply by frinkifail** [reply]
how do I make the original flet app (`main.py`) start `another_app.py` automatically when visiting a specific route?

**Reply by TobiasHT5** [reply]
> how do I make the original flet app (`main.py`) start `another_app.py` automatically when visiting a specific route?

Why don't you use views in that case instead of starting a new app which might not be ideal.

**Reply by frinkifail** [reply]
well that defeats the point I'm trying to make here

I want to render an existing app inside of a flet app (for example if I have a large project and I want to render it inside another project, I don't have the time to convert everything to work with views)

so, how do I make the original app run the other app then?
